### DISCUSSION TITLE
Copilot Autofix for Dependabot

### DISCUSSION THREAD

**Discussion by carlincherry** [community/community#141502] [Other Feature Feedback, Questions, & Ideas]
GitHub Universe 2024 Session: [Tackling supply chain vulnerabilities with Dependabot](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1724774677514001aZun)

Dependency updates can introduce breaking changes that lead to failing CI tests and deployment delays. Identifying the exact cause of these breaks and implementing the correct fix can require significant time and effort, making it challenging to stay on the most up-to-date and secure version of a dependency.

**Dependabot can now leverage the power of Copilot Autofix to analyze dependency updates that fail CI tests and suggest fixes, all within the pull request.** Copilot Autofix for Dependabot not only helps keep your dependencies up to date, but also keeps your CI green. Staying up-to-date on dependencies upgrades with breaking changes is now easier and faster than ever.

If you have any questions, feedback, or ideas regarding the feature, whether you attended Universe in-person or virtually, please add them in the comments below! To learn more, please check out the [changelog](https://github.blog/changelog/2024-10-29-copilot-autofix-for-dependabot-now-available-for-typescript-repositories-private-preview/) and sign up for the [feature waitlist](https://github.com/features/preview/copilot-autofix-for-dependabot).

**Comment by decyjphr** [comment]
Hi @carlincherry , Do you have screenshots or a demo of this? Thank you

**Reply by carlincherry** [reply]
Hi! Can you email me at carlincherry@github.com? Let's chat 😸

**Reply by Kamran21202** [reply]
Hi

**Comment by ramblingenzyme** [comment]
Hi @carlincherry, was this feature abandoned? I assume the changelog was pulled because the private preview ended, but I can't find anything about it being shipped.

https://web.archive.org/web/20250728085433/https://github.blog/changelog/2024-10-29-copilot-autofix-for-dependabot-now-available-for-typescript-repositories-private-preview/
